koku/api/report/provider_map.py

```diff
@@ -95,6 +95,11 @@ def tag_column(self):
         default = self._provider_map.get("tag_column")
         return report_specific_column if report_specific_column else default
 
+    @property
+    def aws_category_column(self):
+        """Return the appropriate query table for the report type."""
+        return self._provider_map.get("aws_category_column")
+
```

Comment:
I feel like we could probably find a way to avoid having to do this in the base class here? Potentially somehow with [getattr](https://docs.python.org/3/library/functions.html#getattr)?

Everything else in here applies to every provider map whereas this one would be specific to AWS, it would be good to try and avoid provider specifics here if we can

Reply:
Switched to using "the walrus"